cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -592,6 +601,7 @@ func (s *Server) Index(args *indexArgs) (state indexState, err error) {
 		case build.IndexStateMeta:
 			log.Printf("updating index.meta %s", args.String())
 
+			// TODO(stefan) handle mergeMeta for tenant id.
```

Comment:
:O also need exploding then to understand the new naming scheme?

Reply:
This here is a bit different. We should probably trigger a reindex if the tenant id changes, but I didn't want to include it now because that would trigger a reindex for everyone.

For explode, I updated the naming gated by tenant enforcement. I will file a follow-up to make that nicer. It's obviously too brittle.